raiden/connection_manager.py

```diff
@@ -257,11 +259,14 @@ def _open_and_deposit(self, partner, funding_amount):
                 token_address=pex(self.token_address),
             )
         else:
-            self.api.deposit(
-                self.token_address,
-                partner,
-                funding_amount,
-            )
+            try:
+                self.api.deposit(
+                    self.token_address,
+                    partner,
+                    funding_amount,
+                )
+            except (TransactionThrew, AddressWithoutCode):
+                pass
```

Comment:
I would recommend some `log.warn()` here

Reply:
I'm not 100% sure this is the correct change, @konradkonrad what do you think?

Reply:
If you don't care about the transaction result or to con, you could add a `wait` parameter and avoid polling/checking/throwing on it.

Reply:
@hackaugusto: Why would you want to silently ignore the deposit transaction throwing? Or the channel or token contract having no code?

Reply:
This is not that serious because it's a channel opened by the connection manager, it's not a bug as it is quite probable that the selected target node is leaving the network.

I'm fine with adding the logging btw, but I want first know what is a good fix from the connection manager perspective. Since the channel closed while it was being opened the number of partners will be off, I'm not sure what are @konradkonrad plans for this, perhaps a background task that will open channels as required?

Reply:
> This is not that serious because it's a channel opened by the connection manager, it's not a bug as it is quite probable that the selected target node is leaving the network.

You are describing the `AddressWithoutCode` exception being thrown I guess because of a closed channel. This can be just one interpretation, it could be anything really. A logging would help us figure out that something even happened. 

Also the `TransactionThrew` exception could happen due to a number of reasons, e.g.: insufficient ether for gas, and since you are also catching that one perhaps either make 2 `log.warn()` or make the `log.warn()` a bit more generic?

Reply:
done